Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169381,'Experiment','RNA binding activity of the recessive parkinsonism protein DJ-1','Parkinson disease (PD) is a major neurodegenerative condition with several rare Mendelian forms. Oxidative stress and mitochondrial function have been implicated in the pathogenesis of PD but the molecular mechanism(s) involved in the degeneration of specific neuronal groups remains unclear. DJ-1 mutations are one cause of recessive parkinsonism, but this gene is also involved in cancer by promoting Ras signaling and suppressing PTEN-induced apoptosis. The specific function of DJ-1 is unclear, although it is responsive to oxidative stress and may play a role in the maintenance of mitochondria. Here we show that DJ-1 associates with specific RNA targets in cells and in the brain including mitochondrial genes, genes involved in glutathione metabolism and members of the PTEN/PI3K cascade. Pathogenic recessive mutants are deficient in this activity. We show that DJ-1 is sufficient for RNA binding at nanomolar concentrations in vitro and that there is some RNA sequence specificity to the association. Oxidative stress causes DJ-1 to dissociate from RNA. Using in vitro and in vivo models of mild oxidative stress, we show that DJ-1 normally suppresses translation in normal circumstances but allows translation after oxidative stress. We tested the hypothesis that these specific RNA targets are responsible for sensitivity to stress by exposing knockout flies to glutathione synthesis inhibitors and saw the predicted increased sensitivity in vivo. These data implicate a single mechanism for the pleiotropic effects of DJ-1 in different model systems, namely that the protein binds and regulates specific groups of RNA targets in an oxidationdependent manner. Furthermore, these results suggest how a small protein might both be an upstream regulator of processes important in parkinsonism and be a modifier of cancer-related processes.','6-8 biological replicates of RNA immunoprecipitated with DJ-1 protein was compared to RNA isolated with a nonspecific IgG control antibody in human cells, or to nonspecific RNA isolated with the DJ-1 antibody from KO mouse brains.',null,null,null,'J Raphael Gibbs','Omicsoft: GSE8632',null,'GPL2507; GPL4234','GSE8632',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169382,'Experiment','Transcriptional Profiling of Subjects from the Iowa Adoption Studies','Transcriptional profiling has been used to identify gene expression patterns indicative of general medical illnesses such as atherosclerosis.    However, whether these methods can identify common psychiatric disorders has not been established.  To answer this question with respect to nicotine use, we used genome wide expression profiling lymphoblast cell lines from six actively smoking Iowa Adoption Studies subjects and nine ?clean? control subjects, followed by real-time PCR (RT-PCR) of gene expression patterns in lymphoblast derived RNA from 94 subjects in the Iowa Adoption Studies.  As compared to those from controls without a history of smoking (n=9), the expression levels of 579 of 29,098 genes were significantly up-regulated and expression levels of 584 of 29,098 genes were significantly down-regulated in lymphoblast lines from currently smoking subjects (n=6).  RT-PCR confirmation of four select RNA levels confirmed the validity of the overall profile and revealed highly significant relationships between the expression of some of these transcripts and 1) major depression, 2) antisocial personality, 3) nicotine dependence and 4) cannabis dependence.  We conclude that the use of expression profiling may contribute significant insights into the biology of complex behavioral disorders.','case vs control',null,null,null,'Gi-yung Ryu','Omicsoft: GSE6264',null,'GPL1426','GSE6264',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169383,'Experiment','Genome-wide gene expression in HDAC5 wildtype and knockout mice treated with chronic cocaine or saline','Some individuals are more vulnerable than others to chronic psychiatric illnesses, such as drug addiction or depression, but the neural and molecular mechanisms responsible for this are poorly understood.  Although addiction and depression are known to have a strong genetic component, recent evidence has suggested that epigenetic mechanisms, such as histone acetylation, may also contribute.  However, the mechanisms which integrate drugs of abuse or stress with changes in chromatin structure remain unclear.  Here, we identify the activity-dependent class II histone deacetylase, HDAC5, as a central integrator of these stimuli with changes in chromatin structure and gene expression.  We found that chronic, but not acute, exposure to cocaine or stress decreases HDAC5 function in the nucleus accumbens (NAc), a major brain reward region, which allows for increased histone acetylation and transcription of HDAC5 target genes.  This regulation is behaviorally important, as loss of HDAC5 causes hypersensitive responses to chronic, not acute, cocaine or stress.  Coupling gene expression microarrays and several bioinformatic approaches, we have gone on to identify the most likely HDAC5 target genes responsible for the behavioral maladaptations.  One HDAC5 target, the Substance P Receptor, is upregulated in HDAC5 knockout mice, has increased histone acetylation on its promoter, and modulates cocaine reward, as measured by conditioned place preference.  These findings suggest that proper balance of histone acetylation is a crucial factor in the saliency of a given emotional stimulus, and that disruption of this balance is involved in the transition from normal adaptive responses to chronic psychiatric illness.','HDAC5 knockout mice and their wildtype littermate controls were injected i.p. with saline or 20 mg/kg cocaine for 7 days.  Twenty-four hours after the last injection, mice were sacrificed and their NAc were dissected.  NAc from four mice were pooled.  Total RNA was extracted from each pool, labeled, and hybridized to Illumina Mouse 6 V1.1 full genome microarrays.  For the first 12 microarrays, 8 mice/condition (WT saline, WT cocaine, KO saline, KO cocaine), were pooled into 2 groups, resulting in 4 mice per array and a total of two biological replicates and four technical replicates.  For the second 18 microarrays, a newly treated set of 16 mice/condition were pooled into 4 groups for a total of 4 mice per array.   This resulted in 4 more biological replicates per condition and 2 more technical replicates.  In total, 30 microarrays yielded 6 biological replicates and 6 technical replicates (1-2 technical replicates per condition).  The goal of this experiment was to identify gene expression differences between HDAC5 knockout and wildtype mice that may underlie their behavioral differences in response to cocaine.',null,null,null,'Will Renthal','Omicsoft: GSE9134',null,'GPL4234','GSE9134',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169384,'Experiment','Rnf41 is associated with anxiety like behavior, major depression and beta carboline induced seizure','The molecular etiology of invididual differences in complex behavior traits and susceptibility to psychiatric illness remains incomplete. Using an unbiased genetic approach in a mouse model, Quantitative Trait Loci (QTL) influencing anxiety-like behaviors and beta-carboline-induced seizure vulnerability have been mapped to the distal portion of mouse chromosome 10 and an interval specific congenic strain (ISCS; A.B6chr10; 66 cM to telomere) was developed. This A.B6chr10 strain facilitated defining the behavioral influences of this region as well as gene expression profiling to identify candidate gene(s) underlying this QTL. By microarray studies, an unsuspected E3 Ubiquitin Ligase, Ring Finger 41 (Rnf41 / Neuregulin Receptor Degrading Protein1; Nrdp1) was differentially expressed in the region of interest, comparing the hippocampi of A/J vs A.B6chr10 mice as well as A/J vs B6 mice. By RT-PCR, Rnf41 expression levels were significantly increased 1.5 and 1.3-fold in the hippocampi of C57BL6/J and A.B6chr10 mice compared to A/J mice, respectively. In addition, protein levels of Rnf41 were increased in hippocampi of B6 mice compared to A/J mice across postnatal development with a 5.5-fold difference at P56. Among LxS recombinant inbred mice (N=33), Rnf41 hippocampal mRNA expression levels were significantly correlated with open field behavior (r= .454, p=.0073). Re-analyzing a microarray database of human post-mortem prefrontal cortex (Brodmann’s Area 46/10), RNF41 mRNA expression levels were reduced significantly in patients with major depression and bipolar disorder compared to unaffected controls. Overall, Rnf41 is a pleiotropic candidate gene for anxiety-like behaviors, depression, and vulnerability to seizures. RNF41 and its binding partners provide novel etiological pathways for influencing behavior, highlighting a potential role for the ubiquitin proteasome system in psychiatric illness.','Microarray. Affymetrix GeneChip: From three independent pools (3 hippocampi /pool) of total RNA for each line, probe was labeled, hybridized as one pool per GeneChip, using a total of six mouse MOE 430 arrays (Affymetrix, Santa Clara, CA). The sequential steps of reverse transcriptions, purification of ds cDNA, in vitro transcription and labeling of cRNA with biotin (Enzo BioArray High Yield RNA Transcript Labeling Kit), fragmentation of cRNA, hybridization to gene chips in a Affymetrix fluidics station 400 system, staining with streptavidin-phycoerythrin, washings, and scannings with a GeneArray Scanner were performed as described in the detailed protocols of the Affymetrix GeneChip Expression Analysis Technical Manual (Affymetrix, Santa Clara, CA) by the UTSW Microarray Core. The Affymetrix two GeneChip MOE 430 A and B set contains more than 45,000 probe sets, interrogating the expression level of 34,000 mouse gene transcripts.. Illumina Microarray;For the P0.5 microarray study, total RNA from a whole brain was individually used per Sentrix Mouse-6 BeadChip (Illumina, San Diego, CA), allowing individual samples to be evaluated and an increased sample size, namely six P0.5 mouse brains for each of three lines (A/J, B6, A.B6chr10). The Sentrix Mouse-6 BeadChip contains total 47,769 gene-specific, 70-mer oligonucleotide probes. Total RNA from each brain tissue of P0.5 mice was extracted in Trizol as above, reverse transcribed, labeled, and hybridized with a microarray according to the manufacturer’s detailed protocol (Illumina, San Diego, Ca.). Briefly, total RNA was reverse transcribed, then in vitro transcribed to cRNA and labeled with biotin-16-UTP. The labeled probe was hybridized to Illumina''s Sentrix mouse-6 Expression BeadChips (Illumina, San Diego, CA), washed, and scanned with Illumina''s BeadStation 500GX Genetic Analysis System in the UTSW Microarray core.',null,null,null,'Sanghyeon Kim','Omicsoft: GSE8641',null,'GPL4055','GSE8641',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169385,'Experiment','GSTM3 single nucleotide polymorphism (rs7483) associated with peripheral BMC expression in Alzheimer disease','Comparisons between the sample groups (normal elderly control (NEC) and Alzheimer disease (AD)) allowed the identification of genes with disease expression patterns associated with the glutathione S-transferase M3 single nucleotide polymorphism rs7483.','Peripheral blood mononuclear cells (BMC) were obtained from normal elderly control (NEC) and Alzheimer disease (AD) subjects.  Targets from biological replicates of NEC (n=18) and AD (n=16) were generated and the expression profiles were determined using the NIA Human MGC cDNA microarray.',null,null,null,'Olivier Charles Maes','Omicsoft: GSE4227',null,'GPL1211','GSE4227',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (169386,'Experiment','Alzheimer Disease peripheral blood mononuclear cell expression','Comparisons between the sample groups (normal elderly control (NEC) and Alzheimer disease (AD)) allow the identification of genes with disease and gender expression patterns.','Peripheral blood mononuclear cells were obtained from normal elderly control (NEC) and Alzheimer disease (AD) subjects.  Targets from biological replicates of female (n=7) and male (n=7) NEC and  female (n=7) and male (n=7) AD were generated and the expression profiles were determined using the NIA Human MGC cDNA microarray.',null,null,null,'Olivier Charles Maes','Omicsoft: GSE4226',null,'GPL1211','GSE4226',to_timestamp('02-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
Insert into biomart.BIO_EXPERIMENT (BIO_EXPERIMENT_ID,BIO_EXPERIMENT_TYPE,TITLE,DESCRIPTION,DESIGN,START_DATE,COMPLETION_DATE,PRIMARY_INVESTIGATOR,CONTACT_FIELD,ETL_ID,STATUS,OVERALL_DESIGN,ACCESSION,ENTRYDT,UPDATED,INSTITUTION,COUNTRY,BIOMARKER_TYPE,TARGET,ACCESS_TYPE) values (783010,'Experiment','Alteration of gene expression profile in HEK293 cells treated with proteasome inhibitor epoxomicin','The objective of this study is to identify the genes that are up-regulated amid proteasome dysfunction to facilitate the discovery of proteolytic pathways that are activated as a compensatory response to proteasome inhibition. Proteasome is a large multi-component proteolytic complex in the cell. It is responsible for the constitutive turn-over of many cellular proteins as well as the degradation of oxidized and/or unfolded proteins. With such a fundamental role in the cell, disruption of proteasome understandably can lead to disastrous outcome. Oxidative stress has been postulated as the driving mechanism for aging. Oxidatively modified proteins, which usually have lost their activity, require immediate removal by proteasome to maintain normal cellular function. Dysfunction of proteasome has also been linked to neuro-degenerative diseases such as Alzheimer’s and Parkinson’s diseases, those that are most commonly seen in aged population. There is more than one proteolytic pathway in the cell, and it has been reported that obstruction of any one of these pathways may enhance the activity of the others. Proteasomal function has been found to have decreased during aging, prompting researchers to hypothesize that failure to remove oxidized proteins may play an important role in aging. It would be interesting to determine the other proteolytic pathways that are activated after proteasome inhibition by a relatively specific inhibitor epoxomicin to help understand their roles in aging processes.','Human embryonic kidney cells (HEK293) were untreated or treated with 1 uM epoxomicin for 1 and 6 hours. Total RNA was extracted for gene expression profiling using Affymetrix human genome U133 2.0 plus chip. A quadrupliate was prepared for each treatment.. contributor: NHLBI Gene Expression Core Facility',null,null,null,'Allen HK Chang','Omicsoft: GSE14429',null,'HG-U133_Plus_2','GSE14429',to_timestamp('03-MAR-12','DD-MON-RR HH.MI.SSXFF AM'),null,null,null,null,null,null);
